src/index.coffee.md

```diff
@@ -10,13 +10,18 @@ information.
 
 ## Usage
 
-Callback approach, for ease of use:   
+Direct synchronous call that returns full result set:
 
-`stringify(data, [options], callback)`   
+`const stringify = require('csv-stringify/lib/sync')`
+`stringify(records, [options]`
 
-Stream API, for maximum of power:   
+Callback approach, for ease of use (note that this returns full result set):
```

Comment:
I assume you're using `require('csv-stringify')` and not `require('csv-stringify/lib/sync')`, in such case, it return a readable stream and not the result set.